packages/core/src/http-client/http-client-types.ts

```diff
@@ -74,3 +74,8 @@ export interface HttpResponse extends KnownHttpResponseFields {
 export interface HttpReponse extends KnownHttpResponseFields {
   [otherKey: string]: any;
 }
+
+export interface HttpRequestAndResponse{
+  request: HttpRequest;
+  response: HttpResponse;
```

Comment:
Seconde discussion @marikaner @jjtang1985 @florian-richter related to the one above. For the opanApi client we return an `AxiosResponse` if you execute `executeRaw` which has a data field and sone other information. I wonder if we could align the two `executeRaw`. For me it is an implementation detail that we use axios so I would rather have a return type which could orientate at the `AxiosResponse` but perhaps also includes the request.

Reply:
For the missing alignment, I was about to add a ticket to add the original request to the openapi one as well.

Reply:
Fine, I guess it is conceptional not possible to remove the deserializing of the response like in the `executeRaw`, because this is done within the generated client where we can not do much?

Reply:
Although we cannot influence the underlying openapi magic, but it does not really do so much. For the deserialization it simply returns the code below with the return type `AxiosPromise<Array<TestEntity>>>`
```
return axios.request(axiosRequestArgs);
```
So with the current version, we are still able to achieve the `executeRaw`.